client/my-sites/people/invite-people/index.jsx

```diff
@@ -270,6 +271,18 @@ const InvitePeople = React.createClass( {
 	},
 
 	render() {
+		if ( this.props.site && ! get( this.props, 'site.capabilities.promote_users' ) ) {
+			return (
```

Comment:
It might be nice to wrap this up in its own function so that render is not that long.